stock_inventory_verification_request/models/stock_slot_verification_request.py

```diff
@@ -0,0 +1,136 @@
+    _name = 'stock.slot.verification.request'
+    _inherit = 'mail.thread'
+
+    @api.model
+    def create(self, vals):
+        vals['name'] = self.env['ir.sequence'].next_by_code(
+            'stock.slot.verification.request') or ''
+        return super(SlotVerificationRequest, self).create(vals)
+
+    @api.multi
+    def _count_involved_moves(self):
+        for rec in self:
+            rec.involved_move_count = len(self.involved_move_ids)
+
+    @api.multi
+    def _count_involved_inv_lines(self):
+        for rec in self:
+            rec.involved_inv_line_count = len(self.involved_inv_line_ids)
+
+    name = fields.Char(string='Name', readonly=True)
+    inventory_id = fields.Many2one(comodel_name='stock.inventory',
+                                   string='Inventory Adjustment',
+                                   readonly=True)
+    inventory_line_id = fields.Many2one(comodel_name='stock.inventory.line',
+                                        string='Inventory Line',
+                                        readonly=True)
+    location_id = fields.Many2one(comodel_name='stock.location',
+                                  string='Location',
+                                  required=True)
+    state = fields.Selection(selection=[
+        ('wait', 'Waiting Actions'),
+        ('open', 'In Progress'),
+        ('cancelled', 'Cancelled'),
+        ('done', 'Solved')
+    ], string='Status', default='wait')
+    responsible_id = fields.Many2one(comodel_name='res.users',
+                                     string='Assigned to')
+    product_id = fields.Many2one(comodel_name='product.product',
+                                 string='Product', required=True)
+    notes = fields.Text('Notes')
+    involved_move_ids = fields.Many2many(
+        comodel_name='stock.move',
+        relation='slot_verification_move_involved_rel',
+        column1='slot_verification_request_id',
+        column2='move_id',
+        string='Involved Stock Moves')
+    involved_move_count = fields.Integer(compute=_count_involved_moves)
+    involved_inv_line_ids = fields.Many2many(
+        comodel_name='stock.inventory.line',
+        relation='slot_verification_inv_line_involved_rel',
+        column1='slot_verification_request_id',
+        column2='inventory_line_id',
+        string='Involved Inventory Lines')
+    involved_inv_line_count = fields.Integer(compute=_count_involved_inv_lines)
+
+    @api.model
+    def _get_involved_moves_domain(self):
+        domain = [('product_id', '=', self.product_id.id), '|',
+                  ('location_id', '=', self.location_id.id),
+                  ('location_dest_id', '=', self.location_id.id)]
+        return domain
+
+    @api.model
+    def _get_involved_lines_domain(self):
+        domain = [('product_id', '=', self.product_id.id),
+                  ('location_id', '=', self.location_id.id)]
+        return domain
+
+    @api.model
+    def _get_involved_lines_and_locations(self):
```

Comment:
If you change the two previous methods to `@api.multi`, then you should change this one too.